app/controllers/users_controller.rb

```diff
@@ -41,11 +41,11 @@ def get_all_users
   end
 
   def get_users
-    User.select("users.id, users.user_name")
+    User.select("users.id, users.user_name").order(:user_name)
   end
 
   def user_todos_number
-    User.all.collect do |user|
+    get_users.collect do |user|
       TodoList.where(:user_id => user.id).count
```

Comment:
1) Qani ver stx prosto `count` stanumnq, `order` petqa ?

2) Mindel karinq sti aninq `TodoList.where(:user_id => user.id).count` > `user.todo_lists.count`.

Reply:
1. `render`-i `todos_number` - y, ver uxarkvuma client side, ta integerneri arraya, vortex amen integer asuma, ta amen user qani todo one. Isk `render`-i `users`-y ta lox usernenenn elya arrayi mech. Meheng, `todos_number`-i 0-errord indexi tivy asuma, ta `users`-i 0-errord indexi usery qani todo one, `todos_number`-i 1-in indexi tivy asuma, ta `users`-i 1-in indexi usery qani todo one, ev aydpes sharunak. Tra heti `order`-y ogtagorcalum.

Reply:
@itc02 
1) Uremns piti indexy poxis, ver srazu todos_numberav get anve.
**TODO-10** PR-umy commentum kiral et pahavu

2) Qani ver `orders` anum erevi et patcharov stachvumchi